### ISSUE TITLE
Not Found errors on mediatypes.appspot.com

### ISSUE THREAD

**Issue by stain** [edsu/mediatypes#2]
http://purl.org/NET/mediatypes/image/png and even http://mediatypes.appspot.com/ fails with

> Error: Not Found

**Comment by edsu** [comment]
Yes, I guess this is because Google no longer supports the crufty old app. I'd like to transition the site to a static site on github, and update the PURLs. But I'm not sure if it's really necessary...

**Comment by wickr** [comment]
We're using the PURLs in OregonDigital.org, it'd be great to have them back up even if static on GitHub. Not sure if this is something I could help with.

**Comment by wickr** [comment]
@edsu Hi Ed. Do you know if there has been any recent development related to this? Thanks.

**Comment by edsu** [comment]
None I'm afraid. It's not s high priority thing for me. If someone wanted to recreate the site and have the PURLs redirect to it they would be fine with me.

**Comment by akuckartz** [comment]
I thought that the 'P' stands for "Persistent".

**Comment by edsu** [comment]
Hahah. Yeah. 

There's more to persistence than calling something persistent :-)

**Comment by wickr** [comment]
@edsu Thanks, looks like SPAR Ontologies has a similar site set up: http://www.sparontologies.net/mediatype

Logins on purl.org seem to be disabled still.

**Comment by wickr** [comment]
We switched over to the SPAR Ontologies mediatype URIs and they are working well.
